From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Remove EDAC code from the build (#146529)
Patch-mainline: obsolete

The more people look at this code, the worse it looks.  It's also
responsible for at least one reported oops in our bug tracking system
(#146529), so let's just disable it from the build entirely.

Also, it's userspace API is going to be radically changing, so we do NOT
want to have to support the old one as it is broken too.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


drivers/Kconfig |    2 --
 1 files changed, 2 deletions(-)

--- linux-2.6.15.orig/drivers/Kconfig	2006-03-10 08:11:57.000000000 -0800
+++ linux-2.6.15/drivers/Kconfig	2006-03-10 08:15:18.000000000 -0800
@@ -68,8 +68,6 @@
 
 source "drivers/sn/Kconfig"
 
-source "drivers/edac/Kconfig"
-
 source "drivers/dma/Kconfig"
 
 endmenu
